app/assets/images/payment_icons/sepa_bank_transfer.svg

```diff
@@ -0,0 +1 @@
+<svg width="38" height="24" viewBox="0 0 38 24" fill="none" xmlns="http://www.w3.org/2000/svg" role="img" aria-labelledby="pi-sepa_bank_transfer"><title id="pi-sepa_bank_transfer">SEPA Bank Transfer</title><rect width="38" height="24" rx="2" fill="#fff"/><rect x=".5" y=".5" width="37" height="23" rx="1.5" stroke="#000" stroke-opacity=".07"/><path d="M28.37 15l-.305-1.037h-2.13L25.647 15h-1.65l1.78-6h2.406l1.803 6H28.37zm-1.368-4.806l-.726 2.6h1.473l-.747-2.6zm-3.86 3.217c-.07.012-.367.023-.89.031l-.265.01h-.95V15h-1.595V9h2.97c.409 0 .72.034.932.102.49.155.817.472.986.952.087.25.13.63.13 1.136 0 .61-.048 1.046-.143 1.313-.19.526-.583.83-1.176.908zm-.377-2.774c-.057-.172-.194-.277-.412-.314a2.902 2.902 0 00-.309-.01h-1.007v1.746h.993c.316-.011.508-.037.577-.074.094-.053.157-.158.189-.316.02-.105.03-.26.03-.468 0-.253-.02-.441-.06-.564zM11.747 14.93c-.242.046-.611.069-1.11.069-.83 0-1.41-.05-1.737-.15-.401-.124-.665-.374-.792-.75-.072-.21-.108-.559-.108-1.046h1.484v.124c0 .261.074.429.222.503a.86.86 0 00.37.082h.545c.279 0 .457-.016.533-.045a.434.434 0 00.269-.262c.023-.075.034-.17.034-.29 0-.264-.095-.424-.286-.482-.07-.023-.402-.054-.991-.095a11.956 11.956 0 01-.988-.1c-.486-.092-.81-.28-.971-.568-.142-.244-.214-.613-.214-1.107 0-.377.04-.679.116-.905.076-.227.2-.399.371-.517.25-.181.57-.287.96-.318.36-.03.72-.045 1.08-.044.622 0 1.066.036 1.33.108.647.175.969.667.969 1.476 0 .066-.005.166-.014.298h-1.484c0-.246-.029-.412-.085-.496-.089-.125-.332-.187-.73-.187-.389 0-.646.036-.77.107-.126.072-.189.229-.189.47 0 .22.056.363.167.432.08.048.185.077.317.086l.3.021c.64.043 1.04.074 1.2.091.507.052.874.188 1.103.41.18.172.286.398.32.677.02.166.03.355.03.564 0 .483-.046.837-.137 1.062-.166.41-.536.67-1.114.78v.002z" fill="#0054A5"/><path d="M17.14 9.845c-.777 0-1.454.428-1.82 1.065h3.297l-.396.858h-3.174c-.017.178-.013.357.012.533h2.917l-.396.86h-2.197c.376.581 1.023.965 1.758.965.629 0 1.193-.281 1.579-.725v1.053a2.994 2.994 0 01-1.681.512 2.989 2.989 0 01-2.752-1.804h-1.05l.396-.86h.434a2.902 2.902 0 01-.004-.533H13.2l.397-.86h.67a2.987 2.987 0 012.773-1.856 2.995 2.995 0 012.077.831l-.345.747a2.09 2.09 0 00-1.63-.788l-.003.002z" fill="#FEC10E"/></svg>
```

Comment:
Same for other icons, the border radius don't look the same.

https://screenshot.click/02-17-qwevg-xleg6.mp4

Reply:
@tauthomas01 It seems that I used an inside stroke, instead of an outside stroke. This caused the icons to show different border-radi. It's now been resolved in all the PR's.